test/Tests/src/Data/Time/Date_Time_Spec.enso

```diff
@@ -190,7 +190,8 @@ spec_with name create_new_datetime parse_datetime nanoseconds_loss_in_precision=
             time . millisecond . should_equal 0
             time . microsecond . should_equal 0
             time . nanosecond . should_equal 0
-            loose_zone_equal time.zone.zone_id  "Etc/UTC"
+            loose_zone_equal (time.zone.zone_id . take (Last 3) . to_case Case.Upper)  "UTC"
+            loose_zone_equal time.zone Time_Zone.utc
```

Comment:
IIRC this last line won't pass - as I posted:
```
> Date_Time.parse "2020-05-06 04:30:20 UTC" "yyyy-MM-dd HH:mm:ss z" . zone == Time_Zone.utc
>>> false
```
On Windows the `Etc/UTC` timezone is not considered equal to `UTC`.